plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,268 @@
+```
+mongo admin -u admin -p admin --eval "db.createUser({user:'pulp',pwd:'<PULP_PASSWORD>',roles:[{role:'dbOwner', db:'pulp_database'},{ role: 'readWrite', db: 'pulp_database'}]})"
+
+```
+
+### Test it works
+From `katello.example.com` test the mongo DB is accessible:
+```
+mongo --host mongo.example.com -u pulp -p <PULP_PASSWORD> --port 27017 --eval 'ping:1' pulp_database
+```
+If there are no errors we are done with database preparation. 
+
+## Fresh install
+
+### Install katello package
+We assume the box where the Katello server will be installed has hostname `katello.example.com`.
+
+Follow the [documentation](plugins/katello/{{ page.version }}/installation/index.html) to install the `katello` package and **do not run foreman-installer.** We need to use the remote database flags with the installer. Use the following steps once the katello rpm is installed.
+
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for installation.
+
+### Run the installer
+To install and configure Katello we just need to run 
+```
+foreman-installer --scenario katello \
+  --foreman-db-host postgres.example.com \ 
+  --foreman-db-password <FOREMAN_PASSWORD> \
+  --foreman-db-database foreman \
+  --foreman-db-manage false
+  --katello-candlepin-db-host postgres.example.com \
+  --katello-candlepin-db-name candlepin \
+  --katello-candlepin-db-password <CANDLEPIN_PASSWORD> \
+  --katello-candlepin-manage-db false \
+  --katello-pulp-db-username pulp \ 
+  --katello-pulp-db-password <PULP_PASSWORD> \ 
+  --katello-pulp-db-seeds “mongo.example.com:27017” \
+  --katello-pulp-db-name pulp_database
+  --katello-pulp-manage-db false
+```
+
+Note: for more related options and tips on SSL configuration see [Full list of options](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#full-list-of-remote-database-related-options-in-the-installer)
+
+## Migration of existing Katello
+We assume that Katello was installed and is running on `katello.example.com`.
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for migration.
+
+### Stop the Katello server
+Stop the Katello related services to minimize risk of the data changes during the migration
+```
+katello-service stop
+systemctl start postgresql mongod
+```
+
+### Dump databases
+Dump the local databases
+```
+foreman-maintain backup online --skip-pulp-content --preserve-directory -y /tmp/migration_backup
```

Comment:
Should this not be an offline backup since we stopped all services?

Reply:
We need the SQL dumps from an online backup (at least that is how we've been testing this process). Foreman-maintain will start the necessary services needed for the online backup (and they are started in the previous step)

Reply:
I guess when I think of an online backup I think of all primary services running. If the webserver is down but databases are up, I don't consider that online given the application is not running and no modifications can be made to the database.

Reply:
The purpose of the online backup is to get the SQL/mongo dumps, since offline will not provide them. We need to use the dumps to seed the remote dbs.  We instruct users to shut down non-database services so nothing is written to the databases at the time of the backup, ensuring an accurate backup.

I'm not sure what action you are looking for here? Can you elaborate on your concerns?